coverage/src/BamView-WebComponent.js

```diff
@@ -120,6 +120,18 @@ class BamViewChart extends HTMLElement {
         this.validBamReadDepth = null;
         upgradeProperty(this, 'label');
 
+        this._regionStart = null;
+        this._regionEnd = null;
+        this._rname = null;
+
+        this._geneStart = null;
+        this._geneEnd = null;
+        this._geneName = null;
+
+        this._regionStart_WG = null;
```

Comment:
This naming isn't proper camel case. Also, it's not obvious what "WG" stands for. I didn't know until I saw the comment below. "Whole genome" means something else. It means every locus of the genome is represented in the data, as opposed to exome which would just be genes represented. Since this view also works for exome files, we should call it something else. I recommend "global view", ie "_regionStartGlobal" etc.